src/event.rs

```diff
@@ -92,6 +92,17 @@ pub enum Event {
     /// will be 0. (This logic is presently per-window but might change to
     /// per-widget to make it more consistent). Otherwise it is in nanoseconds.
     AnimFrame(u64),
+    /// Called on a timer event.
+    ///
+    /// Request a timer event through [`EventCtx::request_timer()`]. That will
+    /// cause a timer event later.
+    ///
+    /// Note that timer events from other widgets may be delivered as well. Use
+    /// the id returned from the `request_timer()` call to filter events more
+    /// precisely.
+    ///
+    /// [`EventCtx::request_timer()`]: struct.EventCtx.html#method.request_timer
+    Timer(usize),
```

Comment:
I would prefer a `TimerToken` type.

Reply:
I can make it an alias or a newtype. (preference?)

I would prefer the concrete type to match Windows though, otherwise we have to do translation.

Reply:
I'd prefer a newtype; and I don't think we should necessarily be making decisions in `druid` based on the implementation details of a single platform.

Reply:
to clarify: we should pick the best reasonable abstraction that can be implemented well across multiple backends, and we should be okay leaning on Rust is Fast if we need to map between a general and a platform representation of some token.

Reply:
One advantage of using a "small integer" is that it can be used as an index of a Vec if you need to maintain a map. But I'm willing to write that off as premature optimization.

It feels to me like the newtype should derive Ord, Hash, Clone, Debug, PartialEq, and Eq, so it can be used as map keys as needed. Sound reasonable?